.github/workflows/buildsCI.yaml

```diff
@@ -535,6 +526,34 @@ jobs:
         python3 unit-tests/run-unit-tests.py --no-color --debug --stdout --not-live --context "dds linux" --tag dds
 
   #--------------------------------------------------------------------------------
+  U-Latest_ST_Py_EX_CfU:  # Ubuntu latest, Static, Python, Examples & Tools, Check for Updates
```

Comment:
Why did you prefer to call it latest instead of specific (as U24)?
If the aim is to avoid revision of this file each 2 years, maybe we can call U-oldest-supported instead of U22 (or something shorter)?

Reply:
I added this to get a heads up from GHA on newest supported Ubuntu.
When Ubuntu 26 enter, we will know from start if we support it, if it will fail we will know to deprecate this test and open a ticket to support it.

It kind of an enhancement for keeping the SDK up to date.
Latest means newest :)
What do you think?